packages/hydrooj/src/handler/misc.ts

```
@@ -139,10 +140,28 @@ export class SwitchAccountHandler extends Handler {
     }
 }
 
+class HeapSnapshotHandler extends Handler {
+    @param('worker', Types.Int)
+    async post({ }, worker: number) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)